### ISSUE TITLE
Unable To View Web Socket Reports

### ISSUE THREAD

**Issue by vakil-neelin** [artilleryio/artillery#1127]
I am on 2.0.0 dev 5 and I cannot open Web Socket JSON reports on `https://reportviewer.artillery.io/` and when I try and use the report function the html file has no graphs on it so its not very useful.

I am able to view HTTP reports with the viewer, but neither have graphs when using the report command.

Is there maybe a dev version of the report viewer?

**Comment by gimos760** [comment]
Hi mate,
I have same question as yours. but mine is Socketio reports, wait for the answer.

**Comment by npopov-d3** [comment]
If you look at console log in debugger there are exceptions thrown:
Uncaught TypeError: Cannot read properties of undefined (reading 'bin')
    at r (metricsgraphics.min.js:1)
    at b (metricsgraphics.min.js:1)
    at e.init (metricsgraphics.min.js:2)
    at new e (metricsgraphics.min.js:3)
    at Object.MG.data_graphic (metricsgraphics.min.js:1)
    at report.html:1648

**Comment by phisch1991** [comment]
Unfortunately, I have got the same issue with artillery/2.0.0-4 darwin-x64 node-v17.0.1. The report shows the raw input, but no graphics. The console shows the same exception as @npopov-d3 posted. Does anyone have a workaround or temporary solution for this (e. g. downgrade to a specific version)?

**Comment by brafales** [comment]
I am experiencing the same issue with the following version:

```
------------ Version Info ------------
Artillery: 1.7.9
Artillery Pro: not installed (https://artillery.io/pro)
Node.js: v14.17.3
OS: darwin/x64
--------------------------------------
```

**Comment by flentini** [comment]
there is a bug in reports in v2, I suggest to downgrade to `1.7.9` until that's fixed. 

@brafales I've just tested reports for both websockets and http in 1.7.9 and they seem to be working fine, can you give me more details about the tests you are running so I can try to reproduce the issue ?

**Comment by hassy** [comment]
fixed in #1203 and available in Artillery `v2.0.0-5`
